<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubwindowsAlignDialog</class>
 <widget class="QDialog" name="SubwindowsAlignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align Windows</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="alignmentGroupBox">
       <property name="title">
        <string>Alignment</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="horizontalRadioButton">
          <property name="text">
           <string>Horizontal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="verticalRadioButton">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gridRadioButton">
          <property name="text">
           <string>Grid</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="conditionGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Condition</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="columnsRadioButton">
          <property name="text">
           <string>Columns</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="columnsSpinBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rowsRadioButton">
          <property name="text">
           <string>Rows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rowsSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="totalSizeGroupBox">
     <property name="title">
      <string>Total Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="WindowSizeEditWidget" name="sizeWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Margins between windows</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="horizontalMarginLabel">
          <property name="text">
           <string>Horizontal:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="horizontalMarginSpinBox">
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="verticalMarginLabel">
          <property name="text">
           <string>Vertical:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="verticalMarginSpinBox">
          <property name="value">
           <number>34</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="hideObjectBrowsersCheckBox">
     <property name="text">
      <string>Hide Object Browsers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindowSizeEditWidget</class>
   <extends>QWidget</extends>
   <header>misc/windowsizeeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SubwindowsAlignDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SubwindowsAlignDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>columnsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rowsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>rowsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>conditionGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
